modules/xmpp/ResumeTask.js

```diff
@@ -127,6 +127,8 @@ export default class ResumeTask {
      * @returns {void}
      */
     _resumeConnection() {
+        this._resumeTimeout = undefined;
```

Comment:
Should you also clear timeout? Like: clearTimeout(this._resumeTimeout); or it is not necessary?

Reply:
Not necessary since this function is called in the timeout handler.